templates/course_info.html

```diff
@@ -1,13 +1,9 @@
-{% extends "bootstrap/base.html" %}
-{% block head %}
-  {{ super() }}
-  <link rel="icon" href="{{ url_for('static', filename='favicon.ico') }}" type="image/x-icon">
-{% endblock %}
-{% block title %}{{ subject }}{% endblock %}
+<!-- {% extends "base.html" %}
+{% block title %} Results {% endblock %}
 
 {% block content %}
 <div class='container'>
-<h2>{{ subject }}</h2>
+<h2>{{ subject|replace('\n', '<br>')|safe }}</h2>
```

Comment:
I suggest the following (although linefeeds are hopefully no longer in the `subject`:
```suggestion
<h2>{{ subject|replace('\n', ' ')|safe }}</h2>
```